OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -34,10 +34,11 @@ public class GravityBombInfo : IProjectileInfo
 
 		[PaletteReference] public readonly string ShadowPalette = "shadow";
 
-		public readonly WDist Speed = WDist.Zero;
+		[Desc("Projectile movement vector per tick. (right, forward, down), negative values for opposite direction.")]
```

Comment:
Can we please make this a local offset (`forward, right, up`) to match turrets, firing offsets, etc?